core/types/src/persistence.ts

```diff
@@ -0,0 +1,33 @@
+export type BaselineCommitment = {
+  // currentHash = H(Underlying data/doc + salt)
+  salt: string;
+  // Values needed for verification (Shield.verifyAndPush inputs)
+  value: string;
+  proof: number[];
+  publicInputs: string[];
+  signatures: object;
```

Comment:
Maybe add a comment that this object will be mapping from participant address -> signature.